### ISSUE TITLE
add 'write_to_cache' and 'read_from_cache' as runtime options

### ISSUE THREAD

**Issue by kcibul** [broadinstitute/cromwell#1695]
Currently we have the ability to turn caching on/off at a cromwell level, or a workflow level, but sometimes there are specific tasks in a workflow that should either participate or not in call caching.

Similar to the workflow options, a task-level runtime set of options should exist to expose this behavior:
write_to_cache: don't add the results of this call to the cache
read_from_cache: don't attempt to use the cache for this call

Requested by Lee as they migrate off Queue

**Comment by cjllanwarne** [comment]
I think this is another thing that we should add to the "what exactly do we want from runtime attributes".

I don't *really* like baking this decision eternally into the WDL file. And maybe this should be call- rather than task-specific.

But I totally agree that we want to have per-call options *somehow*

**Comment by katevoss** [comment]
I agree that this is both a good idea, enabling users to turn on/off caching on a task level, and whether that is a part of the runtime attributes is to be decided. We need to discuss the runtime attributes design separately, so the enabling/disabling may happen elsewhere.

**Comment by katevoss** [comment]
@cjllanwarne this hasn't been implemented, right? 
I'm starting a [Runtime Attributes spec](https://docs.google.com/document/d/1EcsPrmZ6hKtz9vumhdT3357e1jdhljvs6eXXeT6HTaM/edit#), and start collecting all the things we should fix there.

**Comment by katevoss** [comment]
As a **workflow runner**, I want **to be able to select certain tasks to call cache or not call cache on**, so that I can **?**. @LeeTL1220 to help with this

@geoffjentry thoughts on the following?
- Effort: **TBD**
- Risk: **TBD**
- Business value: **TBD**

**Comment by geoffjentry** [comment]
With the add-on that this should *not* find its way into runtime attributes ....

This should be relatively low effort and risk. The risk would be that whenever we don't get something call caching related correct that causes a lot of angst

**Comment by katevoss** [comment]
@geoffjentry Is this at the time of running the workflow, or after the fact (like #1670)?

As a **workflow runner**, I want **to be able to select certain tasks to call cache or not call cache on**, so that I can **avoid reusing bad or old cache results**. 
- Effort: **Small**
- Risk: **Small to Mediume**
  - This should not be a runtime attribute
  - Make sure users don't overuse this feature and eliminate the benefits of call caching (i.e. clearly state when users should use this)
- Business value: **Medium**

**Comment by geoffjentry** [comment]
@katevoss You could get part of the way there with #1670 but not the `read_from_cache` option

**Comment by AlexMTX** [comment]
Hi! A task-level caching control has been introduced recently in the Cromwell 49. That's cool, thanks for the feature!

https://cromwell.readthedocs.io/en/stable/optimizations/VolatileTasks/
The example is using wdl specification 1.0 and it differs from draft-2 when tackling the meta section:
draft-2 waits for a $string as a metadata value
1.0 waits for $meta_value = $string | $number | $boolean | 'null' | $meta_object | $meta_array
https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#metadata-section
https://github.com/openwdl/wdl/blob/master/versions/draft-2/SPEC.md#metadata-section

Hence, a `volatile: true` is not valid for draft-2, because boolean is not a string. Is it possible to adapt the meta section for draft-2 specification too? The majority of our workflows still stick to the draft-2.

**Comment by aednichols** [comment]
draft-2 is in maintenance only mode and does not receive new features.
